Content.Server/_EE/Medical/CPR/CPRTrainingComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+
+namespace Content.Server._EE.Medical.CPR;
+
+[RegisterComponent]
+public sealed partial class CPRTrainingComponent : Component
+{
+    [DataField]
+    public SoundSpecifier CPRSound = new SoundPathSpecifier("/Audio/Effects/CPR.ogg");
+
+    [DataField]
+    public TimeSpan DoAfterDuration = TimeSpan.FromSeconds(3);
+
+    [DataField] public DamageSpecifier CPRHealing = new()
+    {
+        DamageDict =
+        {
+            ["Asphyxiation"] = -6
```

Comment:
This seems more powerful than I would like. This means CPR revives patients in under 30 seconds even from severe asphyxiation.

Also, this does not seem to be respecting the relevant CVar.

Reply:
To be fair, defibs will just cure asphyxiation for free. (IIRC 30 or 40 asphyxiation cured per use, apply ointment for shock as needed.)